From f351f2c77a8fc344fb535a08415c0354eb6354dc Mon Sep 17 00:00:00 2001
From: Kavin <kavin.sp@iMac-0008.local>
Date: Tue, 27 Nov 2012 11:27:16 +0530
Subject: [PATCH 6/8] added break tags

---
 ssamp.html | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ssamp.html b/ssamp.html
index e75e03c..03799cd 100644
--- a/ssamp.html
+++ b/ssamp.html
@@ -4,5 +4,6 @@
 	<title> SAMPLE PAGE FOR CHECKING BRANCHES</title>
 </head>
 <body>
+	<br></br>
 </body>
 </html>
-- 
1.8.0

