From 98aea1ad035a609bdf872e7cbd8d3a5d8daf3716 Mon Sep 17 00:00:00 2001
From: Kavin <kavin.ns@gmail.com>
Date: Thu, 29 Nov 2012 16:46:47 +0530
Subject: [PATCH 8/8] added lnes tag

---
 ssamp.html | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ssamp.html b/ssamp.html
index 03799cd..27971dc 100644
--- a/ssamp.html
+++ b/ssamp.html
@@ -5,5 +5,7 @@
 </head>
 <body>
 	<br></br>
+	<li></li>
+
 </body>
 </html>
-- 
1.8.0

