From 6137564bd0e5fba42439909d4d397e8e4f1863ec Mon Sep 17 00:00:00 2001
From: Kavin <kavin.ns@gmail.com>
Date: Fri, 30 Nov 2012 09:57:42 +0530
Subject: [PATCH 16/18] removed comments from validate.js

---
 validate.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/validate.js b/validate.js
index 0765c28..66b1103 100644
--- a/validate.js
+++ b/validate.js
@@ -1,4 +1,4 @@
-// single line comments
+
 var count =0;  
 function IsNumber(evt)
         {
-- 
1.8.0

